src/media/css/tiles.styl

```diff
@@ -73,6 +73,14 @@ $greater-than-mobile = unquote('(min-width: 330px)');
     }
 }
 
+[data-type--website] .preview-container {
```

Comment:
This seems like a weird choice for the attribute. I think it would be better to give it a value like `data-tile-type="website"`. Or maybe just make it a class?